### ISSUE TITLE
Footnotes not working

### ISSUE THREAD

**Issue by fardeem** [iamacup/react-native-markdown-display#213]
Footnotes seems to be supported but it doesn't seem to work. I'm rendering the following:

```
const content = `
# hello
        
hello world [^1]

[^1]: Book pg. 11 
`;
```

But it renders like this:

<img width="205" alt="image" src="https://github.com/iamacup/react-native-markdown-display/assets/2018374/145b3ccc-f34f-470d-84ab-177b24eab9db">

Am I doing anything wrong?

**Comment by mobynet1** [comment]
It is so nice to see so much attention by the GitHub folks are giving to issues... I have this issue, yet it seems as though GitHub, and Microsoft, by proxy, simply do not care...

**Comment by matheusbaumgart** [comment]
Yeah, no working for me as well. I guess not ALL markdown-it plugins are supported. Can someone confirm?
I see the element if I log it in the rules. But the <Markdown> component doesn't seem to be able to handle it. It basically ignores and doesn't render anything. No erros either.

```
<Markdown
  debugPrintTree
  markdownit={markdownItInstance}
  style={markdownStyles}
  rules={{
    footnote: (node, children, parent, styles) => {
      console.log("FOOTNOTE NODE", node); // expected output

      return (
        <Text key={node.key} style={styles.video}>
          Return custom component for footnote
        </Text>
      );
    },
  }}
>
  {copy}
</Markdown>
```
